app/components/ui/nestable-list-item.js

```diff
@@ -0,0 +1,13 @@
+import Component from '@ember/component';
+
+export default class NestableListItemComponent extends Component {
+  open = false;
+
+  get hasChildren() {
```

Comment:
So these are ember components — not sure if this is reliably going to re-render? Can you refresh me on the rules for classic components, getters, and rendering?